catalog/to-consul/resource.go

```diff
@@ -289,6 +294,11 @@ func (t *ServiceResource) generateRegistrations(key string) {
 		},
 	}
 
+	hostname, err := os.Hostname()
```

Comment:
rather than doing this every time here, lets make this a field on the `ServiceResource` struct:
```
// SyncCatalogPodName is the name of the pod running the sync catalog.
SyncCatalogPodName string
```

You can add the field on line 183 of the `sync-catalog/command.go` file.